cli/cmd/create.go

```diff
@@ -98,9 +100,13 @@ func getConfigFromFolder(folderPath string) ([]connectors.CreateConnectorRequest
 			log.Printf("found unexpected subfolder in folder: %s. This command will not search through it.", filePath)
 			continue
 		}
+		if filepath.Ext(fileInfo.Name()) != ".json" {
```

Comment:
let's not add this to code. at the end an extension is no guarantee for the content. 

for the deployment of our kafka connectors, i propose we ensure to only give in json files

Reply:
@julien-may if this was an internal repo I'd agree, but this one is open-source and we're not the only ones using it. So I'd prefer to keep this open and not restricted to our very own use-case.
I would expect an open-source tool to not blow up as soon as I put an unexpected file in a folder, and to only look for the files it's supposed to parse.